src/main/java/com/slimebot/commands/BugCommand.java

```diff
@@ -2,69 +2,213 @@
+                         
+                    **Additional context**
+                    """);
+            config.setComment("issueTemplate", """
+                    %title% = Title of the issue
+                    %description% = Description of the issue
+                    %steps% = Steps to reproduce the issue
+                    %expected% = Expected behavior
+                    %solution% = Solution idea
+                    """);
+            config.save();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @ApplicationCommandMethod()
+    public void performCommand(SlashCommandInteractionEvent event) {
+        if (!event.getFullCommandName().equals("bug")) return;
+        if (lastBugReport.containsKey(event.getUser().getId())) {
+            LocalDateTime lastReport = lastBugReport.get(event.getUser().getId());
+            if (lastReport.plusMinutes(5).isAfter(LocalDateTime.now())) {
+                event.reply("Du kannst nur alle 5 Minuten einen Bugreport erstellen!").setEphemeral(true).queue();
+                return;
+            }
+        }
+        lastBugReport.put(event.getUser().getId(), LocalDateTime.now());
+
+        TextInput title = TextInput.create("title", "Titel", TextInputStyle.SHORT)
+                .setMinLength(10)
+                .setPlaceholder("Eine kurze präzise Beschreibung des Bugs")
+                .build();
+
+        TextInput howToReproduce = TextInput.create("steps", "Schritte zum Reproduzieren", TextInputStyle.PARAGRAPH)
+                .setMinLength(10)
+                .setPlaceholder("""
+                        1. Gehe zu '....'
+                        2. Klicke auf '....'
+                        3. Scrolle nach unten zu '....'""")
+                .build();
+
+        TextInput textInput = TextInput.create("bug", "Beschreibung", TextInputStyle.PARAGRAPH)
+                .setMinLength(10)
+                .setPlaceholder("Eine ausführliche Beschreibung des Bugs")
+                .build();
+
+        TextInput expected = TextInput.create("expected", "Erwartetes Verhalten", TextInputStyle.PARAGRAPH)
+                .setMinLength(10)
+                .setPlaceholder("Ich erwarte, dass '....' passiert")
+                .build();
+
+        TextInput solution = TextInput.create("solution", "Lösung", TextInputStyle.PARAGRAPH)
+                .setPlaceholder("Ich würde '....' ändern, damit '....' passiert")
+                .setRequired(false)
+                .build();
+
+        Modal modal = Modal.create("bug", "Melde einen Bug")
+                .addActionRow(title)
+                .addActionRow(textInput)
+                .addActionRow(howToReproduce)
+                .addActionRow(expected)
+                .addActionRow(solution)
+                .build();
+
+        event.replyModal(modal).queue();
+    }
+
+    @Override
+    public void onModalInteraction(ModalInteractionEvent event) {
```

Comment:
Use a ModalHandler in a `@WhenFinished` `public void setup(DiscordUtils manager)` method. See how it is done [here](https://github.com/MineKing9534/java-SlimeBot/blob/75e008626725c18d28cb50bce3aac69e50811bbf/src/main/java/com/slimebot/commands/FdmdsCommand.java#L58C3-L58C3).

Reply:
Done